src/Admin/CitizenProjectAdmin.php

```diff
@@ -277,9 +292,84 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
                     ],
                 ],
             ])
+            ->add('creatorFirstName', CallbackFilter::class, [
+                'label' => 'Prénom du créateur',
+                'show_filter' => true,
+                'field_type' => TextType::class,
+                'callback' => function (ProxyQuery $qb, string $alias, string $field, array $value) {
+                    if (!$value['value']) {
+                        return;
+                    }
+
+                    /* @var ProxyQuery|QueryBuilder $qb */
+                    $qb
+                        ->leftJoin(Adherent::class, 'creator', Join::WITH, sprintf('creator.uuid=%s.createdBy', $alias))
+                        ->andWhere(
+                            'LOWER(creator.firstName) LIKE :firstName '.
+                            "OR $alias.createdBy=:uuid"
+                        )
+                        ->setParameter('firstName', sprintf('%%%s%%', mb_strtolower($value['value'])))
+                        ->setParameter('uuid', $value['value']);
+
+                    return true;
+                },
+            ])
+            ->add('creatorLastName', CallbackFilter::class, [
```

Comment:
maybe do juste one filter fied `Createur` for search in last and first name?

Reply:
I think we must stick to the current layout with lastname and firstname in different filter